modules/iap-connector/variables.tf

```diff
@@ -0,0 +1,47 @@
+variable "address_name" {
```

Comment:
As for the other modules, please add a README file with purpose and usage examples.